### DISCUSSION TITLE
Rowy Run Derivative Deploy Not Working

### DISCUSSION THREAD

**Discussion by asiergonio** [AntlerVC/firetable#778] [Support / Q&A] [answered]
Hi @shamsmosowi @harinij, 

Our Rowy Derivative deploy on google cloud is not working. The build fails and the log on Google Cloud Console says: 

```
Error: Invalid compiled functionConfig.ts
    at generateConfig (/home/node/app/build/functionBuilder/compiler/index.js:59:15)
    at async functionBuilder (/home/node/app/build/functionBuilder/index.js:56:29)
    at async /home/node/app/build/index.js:39:22
```

The derivative code is the following: 

```
const derivative:Derivative = async ({row,ref,db,storage,auth})=>{
    return "Hello World";
  }
```

Thanks in advance for your help. Big fan of what you are building at Rowy! :)

**Comment by st-trade** [comment]
In my case, tasks are not even starting to deploy. These is what the console showss
<img width="655" alt="image" src="https://user-images.githubusercontent.com/94030557/187543507-1e63a4df-17ac-4965-a891-6a62f5687c14.png">

**Comment by shamsmosowi** [comment, marked as answer]
Hi @asiergonio could you provide the complete failed build logs, you can find it in the cloud build logs modal in the top right

**Reply by asiergonio** [reply]
Hi @shamsmosowi ,

![image](https://user-images.githubusercontent.com/64319377/187864680-22ff81e5-a593-4389-8b74-8da40c2a8cd3.png)

Did you mean that? It seems that Rowy is not able to create the bucket due to auth issue

Thanks for your help!

**Reply by shamsmosowi** [reply]
hey @asiergonio, this looks like a separate issue, the first error indicates an issue rowy generated code, this is an example of the logs i mentioned [here](https://demo.rowy.io/table/emojify#tableModal=%22cloudLogs%22&cloudLogFilters=%7B%22type%22%3A%22build%22%2C%22timeRange%22%3A%7B%22type%22%3A%22days%22%2C%22value%22%3A7%7D%2C%22buildLogExpanded%22%3A0%7D)
Email me if you would like to jump on a call to go through both issues
